doc/user/command-line.xml

```diff
@@ -955,9 +954,8 @@ prog.c
       <scons_example name="commandline_Variables1">
         <file name="SConstruct" printme="1">
 vars = Variables(None, ARGUMENTS)
-vars.Add('RELEASE', 'Set to 1 to build for release', 0)
-env = Environment(variables = vars,
-                  CPPDEFINES={'RELEASE_BUILD' : '${RELEASE}'})
+vars.Add('RELEASE', default=0)
```

Comment:
Why remove the help text?

Reply:
To introduce concepts a little more gradually... the very next section introduced help, so I figured just defer it to there. Nothing references the help in this example.